components/scream/src/dynamics/homme/dynamics_driven_grids_manager.cpp

```diff
@@ -181,8 +181,8 @@ void DynamicsDrivenGridsManager::build_dynamics_grid () {
     Kokkos::parallel_for(policy, KOKKOS_LAMBDA(const KT::MemberType& team) {
       const Int i = team.league_rank();
 
-      EKAT_KERNEL_ASSERT_MSG(!isnan(lat(i)), "Error! NaN values detected for latitude.");
-      EKAT_KERNEL_ASSERT_MSG(!isnan(lon(i)), "Error! NaN values detected for longitude.");
+      EKAT_KERNEL_ASSERT_MSG(!ekat::impl::is_nan(lat(i)), "Error! NaN values detected for latitude.");
```

Comment:
SCREAM would not build on my laptop. This can indeed happen when using `isnan` (and similar std functions), depending on the way in which stdlib headers are included. Using ekat's wrapper is always the safe choice.